core/scheduler/src/main/resources/application.conf

```diff
@@ -64,6 +64,11 @@ whisk {
     grpc {
       tls = "false"
     }
+    scheduling {
+      stale-threshold = "100 milliseconds"
```

Comment:
Because the value is `FiniteDuration` type, how about `stale-duration-threshold`?
(btw, `check-interval or drop-interval`, from the name, we can know it is a  `FiniteDuration` type)

Reply:
I just thought its value includes the unit so we can easily figure out its type.